src/index.js

```diff
@@ -0,0 +1,43 @@
+// import _ from 'lodash';
+import './style.css';
+
+const tasks = [
+  {
+    discription: 'Write a  ',
+    complated: false,
+    index: 1,
+  },
+  {
+    discription: 'On page  tasks ',
+    completed: false,
+    index: 2,
+  },
+  {
+    discription: 'The list should aalues for each task.',
+    completed: false,
+    index: 3,
```

Comment:
- Please ensure that all objects in your array have the same keys i.e `description, completed, index`, there are some differences due to spelling. Please fix this 👍